### ISSUE TITLE
Can't install 3.6.0: Undefined array key "icon"

### ISSUE THREAD

**Issue by boboldehampsink** [spicywebau/craft-neo#705]
### Bug Description

Seeing this error when installing 3.6.0 from an existing project config:

```
 > applying existing project config ... Exception: Undefined array key "icon" (/app/user/vendor/spicyweb/craft-neo/src/services/BlockTypes.php:423)
```

### Steps to reproduce

1. Install 3.5.16 with project config
2. Upgrade to 3.6.0 in project config
3. Start project again with new project config

### Neo version

3.6.0

### Craft CMS version

4

### Does this issue involve templating, and if so, is eager-loading used?

This is not a templating issue

**Comment by benoitblancquaert** [comment]
I'm having the same problem and it's causing part of the project configuration not to be applied.

**Comment by ttempleton** [comment]
Thanks for reporting this - fixed now in 3.6.1.
